src/engine/guidance/post_processing.cpp

```diff
@@ -1327,8 +1498,8 @@ std::vector<RouteStep> collapseUseLane(std::vector<RouteStep> steps)
             canCollapeUseLane(step.intersections.front().lanes,
                               step.intersections.front().lane_description))
         {
-            const auto previous = getPreviousIndex(step_index);
-            steps[previous] = elongate(std::move(steps[previous]), steps[step_index]);
+            const auto previous = getPreviousIndex(step_index, steps);
+            steps[previous] = elongate(steps[previous], steps[step_index]);
```

Comment:
Must have gotten in here during rebase. I didn't touch this apart from rebasing. Thanks for spotting it.